packages/xo-server/src/api/pci.mjs

```diff
@@ -24,13 +25,17 @@ export const disableDom0Access = defer(async function ($defer, { pcis, disable }
       stopOnError: false,
     }
   )
+  if (!forceReboot) {
+    throw noHostsAvailable()
+  }
```

Comment:
I don't get this. Isn't the error thrown by `rebootHost`?

Reply:
Indeed this was just for test purposes, let me correct it real quick